roles/alertmanager/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: create config dir
+  file:
+    path: "{{ alertmanager_config_dir }}"
+    state: directory
+    mode: "0755"
+
+- name: install config file
+  template:
+    src: templates/alertmanager.yml.j2
```

Comment:
As written, this will make it a little difficult to adjust the AlertManager config without directly modifying the DeepOps role.

Instead of providing a hard-coded config file, can you make this a variable? E.g.,

```
- name: install config file
  template: "{{ alertmanager_config_src }}"
...
```

With `alertmanager_config_src: templates/alertmanager.yml.j2` in `defaults/main.yml`.

This will allow the user to substitute in their own config file without modifying DeepOps, and making subsequent DeepOps release updates more difficult.

Reply:
Thanks for your comments. :)
When I looked at the [Prometheus configuration](https://github.com/NVIDIA/deepops/blob/master/roles/prometheus/tasks/main.yml#L16), I checked that it was configured as a relative path and followed it as it is. However, I agree with your comment and changed the configuration of Prometheus as well.